.github/workflows/nextflow-tests.yml

```diff
@@ -4,12 +4,99 @@ name: 'Nextflow tests'
 on:
   workflow_call:
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event_name }}-${{ github.sha }}
+  cancel-in-progress: true
+
 # Inspired by https://blog.aspect.dev/github-actions-dynamic-matrix
 jobs:
   discover:
     runs-on: ubuntu-latest
     steps:
+      - uses: actions/github-script@v7
+        name: Parse event
+        id: parse-event
+        with:
+          script: |
+            /*
+            We need to determine if the incoming event was from a pull_request
+            or an issue_comment
+            */
+            let apply_fixes = context.eventName == 'issue_comment'
```

Comment:
At the risk of starting a war, we should considering requiring semicolons. No change requested for now, just a thought for later.

Reply:
Totally fair - I took a cue from the [docs](https://github.com/actions/github-script?tab=readme-ov-file#welcome-a-first-time-contributor) _not_ using them, but I do generally fall on the side of being explicit.

Reply:
Gotcha - makes sense. A discussion for later, either way. :)